index.html

```diff
@@ -1,217 +1,359 @@
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <div class="get-my-resume">
+          <button>Get My Resume</button>
+        </div>
+      </section>
+      <section class="about__cards">
+        <div class="img-patholder language">
+          <div class="box-language">
+            <h2 class="title">Languages</h2>
+            <ul class="buttons button-white-bg">
+              <li>ruby on rails</li>
+              <li>Css</li>
+              <li>javascript</li>
+              <li>html</li>
+            </ul>
+          </div>
+        </div>
+        <div class="img-patholder FrameWorks">
+          <div class="box">
+            <h2 class="title">FrameWorks</h2>
+            <ul class="buttons2 button-white-bg">
+              <li>Bootstrap</li>
+              <li>Ruby on rails</li>
+              <li>Rspec</li>
+            </ul>
+            <ul class="buttons3 button-white-bg">
+              <li>Capybara</li>
+              <li>Selenium</li>
+            </ul>
+          </div>
+        </div>
+        <div class="img-patholder skills">
+          <div class="box">
+            <h2 class="title">Skills</h2>
+            <ul class="buttons button-white-bg">
+              <li>Codekit</li>
+              <li>Github</li>
+              <li>Codepen</li>
+              <li>Gitlab</li>
+            </ul>
+            <ul class="buttons3 button-white-bg">
+              <li>Termenal</li>
+            </ul>
+          </div>
+        </div>
+      </section>
     </div>
-    
-</body>
-
-</html>
\ No newline at end of file
+    <footer>
+      <div class="contact__container" id="contact">
+        <h2 class="title">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+        <form
+          action="https://formspree.io/f/myyowwdj"
+          method="POST"
+          class="contact-form"
+          id="form"
+        >
+          <div>
+            <div class="full__name">
+              <input
+                type="text"
+                name="name"
+                id="full-name"
+                size="40"
+                maxlength="30"
+                placeholder="Full Name"
+                required
+              />
+            </div>
+            <div class="email">
+              <input
+                type="email"
+                name="email"
+                id="email"
+                size="40"
+                maxlength="30"
+                placeholder="Email"
+                required
+              />
+            </div>
+            <textarea
+              name="description"
+              id="description"
+              rows="3"
+              cols="41"
+              maxlength="500"
+              placeholder="Enter text here"
+              required
+            ></textarea>
+          </div>
+          <div>
+            <button class="submit-button" type="submit">Get in touch</button>
+          </div>

```

Comment:
- [ ] Kindly add more space between the `Get in touch` button and the social media icons you can use `margin-botton: 46px` 

Your design    |        Figma
---------:|:------------
![your](https://user-images.githubusercontent.com/79058364/165343775-f60ecdc4-5137-45cb-88c3-55988d057c2e.png)  | ![figma](https://user-images.githubusercontent.com/79058364/165343832-5e701854-a71d-4f1f-936f-f68e1f09107e.png)